eng/pipelines/templates/jobs/archetype-sdk-eng-client.yml

```diff
@@ -0,0 +1,85 @@
+parameters:
```

Comment:
I suspect we can just name this eng.yml or something similar as the client in the name isn't correct for these now.

While I agree we probably do want a separate template I worry a little about the duplication.